### DISCUSSION TITLE
logout after fetchUser throw 401 and Token doesn't exist

### DISCUSSION THREAD

**Discussion by apithy-isidro-martinez** [nuxt-community/auth-module#1556] [Q&A]
how can i dispatch logout if fetchUser throw 401?
![CleanShot 2021-09-14 at 10 44 24](https://user-images.githubusercontent.com/74619952/133290049-0e2bb183-f1c7-4a93-91f9-f6a038064803.png)

**Comment by devzom** [comment]
But that's happend becouse it's seems that You haven't sent expired/ invalid token in header or didn't sent it at all?  Have You checked?
